src/pages/questionAnswerViewer/QuestionAnswerViewer.jsx

```diff
@@ -106,7 +109,32 @@ export default function QuestionAnswerViewer() {
     }
   }
 
-  useEffect(() => { fetchAnswers(); }, [user, question_id, answer_id]);
+  function handleAnswerDeleted(response) {
+    if (response.status === 'error') {
+      displayAlert('error', response.message);
+    } else {
+      displayAlert('success', 'Deleted Answer');
+      history.push(`/question/${question_id}`);
+    }
+  }
+
+  useEffect(() => {
+    fetchAnswers();
+  }, [user, question_id, answer_id]);
+
+  async function getNewAnswer() {
+    const response = await API.queryDispatcher.getAnswer(question_id, user.id_token);
+    if (response.status === 'error') {
+      pageStatus.setFailure(response.message);
```

Comment:
This error response doesn't have a message property so nothing happens when we hit this line. It looks like there's a `detail` property with some kind of structured logging. Not sure if we have a component to display that error. If not, we could just put a static error message.

Reply:
I put in a check for if 'message' is missing from an error response, but we should think more on this as we add new ARAs with potentially different error reporting. #41